elvers/__main__.py

```diff
@@ -15,24 +15,26 @@
 import shutil
 import subprocess
 
-from ep_utils.utils import *
-from ep_utils.pretty_config import pretty_name, write_config
-from ep_utils.print_workflow_options import print_available_workflows_and_tools
-from ep_utils.capture_stdout import CaptureStdout
+from .utils.utils import *
+from .utils.pretty_config  import pretty_name, write_config
+from .utils.print_workflow_options import print_available_workflows_and_tools
+from .utils.capture_stdout import CaptureStdout
+
+from . import _program
 
 def build_default_params(workdir, targets):
     defaultParams = {}
     # first, figure out which parts of the pipeline are being run, and get those defaults
-    pipeline_defaultsFile = find_yaml(workdir, os.path.join('ep_utils', 'pipeline_defaults'), 'pipeline_defaults')
+    pipeline_defaultsFile = find_yaml(workdir, os.path.join('utils', 'pipeline_defaults'), 'pipeline_defaults')
```

Comment:
Note that this line, which looks for the default pipeline file using its location in the repository, greatly reduces the utility of packaging elvers, since with this line, the elvers command line utility will not work unless it is run from a copy of the repository (or unless there is a `utils` folder).

A better approach would be to include the pipeline defaults file `utils/pipeline_defaults.yaml` in MANIFEST.in, so that when users install the package, the pipeline defaults are installed alongside the package.

Note that this issue was uncovered by installing elvers, then running the following command line from the `examples/` directory:

```
elvers nema.yaml diffexp --dagpng=dag_multi.png
```

Reply:
(Also... the `ep_utils/` folder is still there in the `make_package` branch, it seems like it was not renamed to `utils`...?

Reply:
on it!

Reply:
is there anything in that `ep_utils` folder? I renamed it to `elvers/utils`, but forgot to remove the `el_utils` include from the `MANIFEST.in`